resources/user_config.ts

```diff
@@ -37,6 +37,7 @@ const ZoneInfo = _ZoneInfo;
 // Convince TypeScript and eslint that these are used.  TypeScript doesn't have a great way
 // to disable individual rules, so this is safer than disabling all rules.
 console.assert(
+  // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
```

Comment:
I wasn't really sure what the best way to handle this would be. Break it apart into 8 different `console.assert` calls?

I think the comment above covers why the eslint disable is required as well.

Reply:
Yeah, I think the comment covers this.  If the individual lines wouldn't need an eslint disable, that seems like one avenue if you like that better, but if they all would need eslint disable this seems best.

Reply:
It only needs an eslint-disable because the variables are being `&&`-ed together. I think breaking this into multiple asserts is better in general since it makes it easier to add/remove them as needed later?

Reply:
Sounds good to me.